llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -523,19 +642,22 @@ struct LoadedDebugInfoSection {
   Error addNode(Expected<MCObjectProxy> Proxy) {
     if (!Proxy)
       return Proxy.takeError();
-    if (auto CURef = DebugInfoCURef::Cast(*Proxy)) {
+    if (auto FixupsRef = DebugInfoFixupsRef::Cast(*Proxy)) {
+      if (CUData.size())
+        return createStringError(inconvertibleErrorCode(),
+                                 "Invalid FixupsRef afte CURef");
+      FixupsData = FixupsRef->getData();
+    } else if (auto CURef = DebugInfoCURef::Cast(*Proxy)) {
```

Comment:
Unfortunately no, the code is tailored to the specific section it is trying to load, because it also verifies the layout of the cas objects under the section. That layout is different for the abbreviation and info sections